test/distributed/test_c10d_nccl.py

```diff
@@ -52,6 +52,11 @@
 
 if not IS_WINDOWS:
     from torch.distributed.optim.functional_sgd import _FunctionalSGD
+    from torch.distributed.optim.functional_adam import _FunctionalAdam
+    _SUPPORTED_OPTIM_MAPPING = {
```

Comment:
Do you think it will be better to move this mapping to `torch.distributed.optim.__init__`? Depending on whether it will be used by users.

Reply:
Yes, it is duplicated in a few places now. Will do it in a follow up PR if you don't mind.